src/encoded/static/components/item-pages/ExperimentSetView.js

```diff
@@ -116,7 +116,8 @@ export default class ExperimentSetView extends WorkflowRunTracingView {
                 tabs.push(FileViewGraphSection.getTabObject(
                     _.extend({}, this.props, { 'isNodeCurrentContext' : this.isWorkflowNodeCurrentContext }),
                     this.state,
-                    this.handleToggleAllRuns
+                    this.handleToggleAllRuns,
+                    width
```

Comment:
Had forgotten to pass this in earlier -- width for provenance graphs.
Allows us to better (re)center workflow graphs in center of screen/viewport when switching between tabs.
Similar change in FileView.js.